modyn/supervisor/internal/grpc_handler.py

```diff
@@ -44,53 +44,27 @@
 )
 from modyn.storage.internal.grpc.generated.storage_pb2_grpc import StorageStub
 from modyn.supervisor.internal.evaluation_result_writer import AbstractEvaluationResultWriter
-from modyn.supervisor.internal.utils import EvaluationStatusReporter, TrainingStatusReporter
-from modyn.trainer_server.internal.grpc.generated.trainer_server_pb2 import CheckpointInfo, Data
-from modyn.trainer_server.internal.grpc.generated.trainer_server_pb2 import JsonString as TrainerServerJsonString
-from modyn.trainer_server.internal.grpc.generated.trainer_server_pb2 import (
-    PythonString,
-    StartTrainingRequest,
-    StartTrainingResponse,
-    StoreFinalModelRequest,
-    StoreFinalModelResponse,
-    TrainerAvailableRequest,
-    TrainerAvailableResponse,
-    TrainingStatusRequest,
-    TrainingStatusResponse,
-)
-from modyn.trainer_server.internal.grpc.generated.trainer_server_pb2_grpc import TrainerServerStub
+from modyn.supervisor.internal.utils import EvaluationStatusReporter
 from modyn.utils import grpc_common_config, grpc_connection_established
 
 logger = logging.getLogger(__name__)
 
 
-class GRPCHandler:
-    # pylint: disable=too-many-instance-attributes
+class GRPCHandler(TrainerServerGRPCHandlerMixin):
 
     # pylint: disable=unused-argument
-    def __init__(
-        self,
-        modyn_config: dict,
-        pipeline_status_queue: Optional[mp.Queue] = None,
-        training_status_queue: Optional[mp.Queue] = None,
-        eval_status_queue: Optional[mp.Queue] = None,
-    ) -> None:
+    def __init__(self, modyn_config: dict) -> None:
+        super().__init__(modyn_config=modyn_config)
         self.config = modyn_config
-        self.pipeline_status_queue = pipeline_status_queue
-        self.training_status_queue = training_status_queue
-        self.eval_status_queue = eval_status_queue
```

Comment:
it's interesting: these fields are actually not used anywhere

Reply:
hmm to be honest, idk why they are here. as you say, probably some leftovers...